README.md

```diff
@@ -0,0 +1,46 @@
+# Flatpak for nomacs
```

Comment:
Do we need it here? In general it duplicates existing documentation.

Reply:
right now I need it : )
- but you're right, I will remove it once I figured out how to correctly install `apddata.xml` etc